tox.ini

```diff
@@ -6,7 +6,7 @@ max-line-length = 120
 
 [pytest]
 DJANGO_SETTINGS_MODULE = enterprise.settings.test
-addopts = --cov enterprise --cov enterprise_learner_portal --cov consent --cov integrated_channels --cov-report term-missing --cov-report xml
+# addopts = --cov enterprise --cov enterprise_learner_portal --cov consent --cov integrated_channels --cov-report term-missing --cov-report xml
```

Comment:
This seems to be breaking the coverage in the CI. Is it intended?

Reply:
Oops! I had disabled the coverage report when running tests locally. I uncommented them now.